apps/project/tests/test_apis.py

```diff
@@ -0,0 +1,375 @@
+            createTag(data: $input){
+                errors {
+                    field
+                    messages
+                }
+                result {
+                    title
+                    description
+                    project{
+                        id
+                    }
+                }
+                ok
+            }
+        }'''
+
+        self.input = {
+            "title": "Test for the tag creation",
+            "description": "This is the description of the tag",
+            "project": self.project.id,
+        }
+
+    def test_valid_tag_creation(self):
+        response = self.query(
+            self.mutation,
+            input_data=self.input,
+        )
+
+        content = json.loads(response.content)
+        self.assertResponseNoErrors(response)
+        self.assertTrue(content['data']['createTag']['ok'], content)
+        self.assertIsNone(content['data']['createTag']['errors'], content)
+        self.assertEqual(content['data']['createTag']['result']['title'],
+                         self.input['title'])
+
+
+class TestUpdateTag(ChronoGraphQLTestCase):
+    def setUp(self):
+        self.tag = TagFactory.create()
+        self.project = ProjectFactory.create()
+        self.mutation = ''' mutation UpdateTag($input: TagUpdateInputType!){
+            updateTag(data: $input){
+                errors {
+                    field
+                    messages
+                }
+                result {
+                    id
+                    title
+                    project{
+                        id
+                    }
+                    description
+                }
+                ok
+            }
+        }'''
+        self.input = {
+            "id": self.tag.id,
+            "description": "This is for the test",
+            "project": self.project.id
+        }
+
+    def test_valid_tag_update(self):
+        response = self.query(
+            self.mutation,
+            input_data=self.input
+        )
+        content = json.loads(response.content)
+        self.assertResponseNoErrors(response)
+        self.assertTrue(content['data']['updateTag']['ok'], content)
+
+
+class TestDeleteTag(ChronoGraphQLTestCase):
+    def setUp(self):
+        self.mutation = '''mutation DeleteTag($id: ID!){
+                deleteTag(id: $id){
+                    errors {
+                        field
+                        messages
+                    }
+                    result {
+                        id
+                        title
+                    }
+                    ok
+                }
+            }'''
+        self.tag = TagFactory.create()
+        self.variables = {
+            "id": self.tag.id,
+        }
+
+    def test_valid_tag_delete(self):
+        response = self.query(
+            self.mutation,
+            variables=self.variables,
+        )
+        content = json.loads(response.content)
```

Comment:
```suggestion
        content = response.json()
```
FYI, you can do this as well.